### DISCUSSION TITLE
Weird CSS and GA implementation

### DISCUSSION THREAD

**Discussion by krzysiekpaszek** [adityatelange/hugo-PaperMod#263] [Q&A]
Hi.
I would like to do 2 things:

1) add Google Analytics code not as a separate file, but directly into the HTML code
2) same with CSS: how to inject final minified CSS code not into a separate file, but into the head of each page?

**Comment by kdkasad** [comment]
You can create an `extend_head.html` file which will have its contents included in the page's `head`: https://adityatelange.github.io/hugo-PaperMod/posts/papermod/papermod-faq/#custom-head--footer.

**Reply by krzysiekpaszek** [reply]
thank you.
But how would I add CSS, this is auto generated.
